.claude/agents/aqa-engineer.md

```
@@ -0,0 +1,196 @@
+- Benchmark 1: {what to measure}
+- Target: {threshold}
+
+**Validation Commands:**
+```bash
+npm run typecheck  # TypeScript strict
+npm run lint       # ESLint 0 errors
+npm run test:unit  # Unit tests >80%
+npm run test:e2e   # E2E tests (if applicable)
+npm run validate   # Asset/license validation
+```
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)